lib/bio_monitor/sensor_manager.ex

```diff
@@ -51,13 +51,23 @@ defmodule BioMonitor.SensorManager do
     end
   end
 
+  def get_ph_offset do
+    with {:ok, value} <- send_and_read(:ph, "getPhOffset"),
+      {:ok, parsed_value} <- parse_reading(value)
+    do
+      parsed_value
+    else
+      _ -> :error
+    end
+  end
+
   @doc """
     sets the offset of the ph sensor for calibration
   """
-  def set_ph_offset(offset) do
-    case send_command(:ph, "setPhOffset:#{offset}") do
+  def set_ph_offset(target, offser) do
```

Comment:
fix the typo or it wont work :)

Reply:
even worse, i was doing string interpolation with the javascript way (using $ instead of #) 😬

Reply:
oh haha i use that in kotlin, so it seemed fine